src/modules/users/user-list/users.js

```diff
@@ -0,0 +1,69 @@
+(function() {
+    "use strict";
+
+    angular
+        .module("TendrlModule")
+        .component("users", {
+
+            restrict: "E",
+            templateUrl: "/modules/users/user-list/users.html",
+            bindings: {},
+            controller: userController,
+            controllerAs: "userCntrl"
+        });
+
+    /*@ngInject*/
+    function userController($scope, $rootScope, $state, $interval, utils, config, userStore) {
+
+        var vm = this,
+            userTimer,
+            userList;
+
+        //vm.userStatus = [];
+        vm.isDataLoading = true;
+
+        init();
+
+        function init() {
+            userStore.getUserList()
+                .then(function(data) {
+
+                    vm.isDataLoading = false;
+                    vm.userList = [];
+                    if (data !== null) {
+                        vm.userList = userStore.setupUserListData(data);
+                    }
+                    startTimer();
```

Comment:
@gnehapk I am not sure why we would need a polling for user's list. Mostly user's list screen should be on demand and this is something which is not used every now and then. @Tendrl/tendrl-core comments?